src/screens/ChannelListScreen/index.tsx

```diff
@@ -25,6 +26,9 @@ const ChannelListScreen = ({route}) => {
   const isFocused = navigation.isFocused();
   const {channels, goToChatScreen, goToPostDetailScreen, goToCommunityScreen, goToContactScreen} =
     useSignedChannelListScreenHook();
+  const ref = React.useRef(null);
+
+  useScrollToTop(ref);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useScrollToTop(ref)` is correct, but it's important to ensure that the referenced component (in this case, `FlatList`) supports scroll functionality. If not, this could lead to unexpected behavior or errors.
```diff
+  const ref = React.useRef(null);
+ 
+  useScrollToTop(ref);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->